### ISSUE TITLE
Design Intent - Accessibility Feedback - IIR, VA.gov Onboarding Flow MVP

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/va.gov-team#115009]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** The modal heading should use the design system’s `title` property, which is an h2 by default and tested for accessibility. Screen readers will announce this correctly. Please use this property instead of a custom h1.
- [ ] **Consider:** Are you planning to open a new modal for each step, or just update the content within the same modal?
  - We recommend opening a new modal for each step, so screen readers will re-announce the content.
  - If you update content in the same modal, you’ll need to manually move focus to the heading or modal container.
- [ ] **Must:** Good job including 200% zoom testing. Make sure to test up to 400% zoom for full accessibility. Set the browser to 1280px wide, then test at 200%, 300%, and 400% zoom.
  - Watch out for images in the modal at high zoom. They may get too small to be useful. You might need alternate images for different zoom levels or screen sizes.
- - [ ] **Should:** This is a stepped modal process and users can’t go back. Is that intentional, or should they be able to return to previous steps? Did you look into alternative ways to deliver this content? If not, we recommend exploring other options, such as having a separate page.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [x] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app
